pkgs/applications/misc/navit/CMakeLists.txt.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 763f75b..defa74a 100755
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -212,7 +212,7 @@ CHECK_INCLUDE_FILES(endian.h HAVE_ENDIAN_H)
+ CHECK_INCLUDE_FILES(stdint.h HAVE_STDINT_H)
+ CHECK_INCLUDE_FILES(byteswap.h HAVE_BYTESWAP_H)
+ CHECK_LIBRARY_EXISTS(gypsy gypsy_control_get_default "" GYPSY_FOUND)
+-CHECK_INCLUDE_FILES(libspeechd.h HAVE_LIBSPEECHD)
```

Comment:
Why is this patch necessary? It should either indicate a packaging failure on our side or an insufficiently flexible build script upstream. Either way it should be fixed.

Reply:
you think we should symlink the headers here like other distros ? i wonder  it was the new place and i've fix it.

Reply:
Are other distros doing that?

Reply:
or copy https://packages.debian.org/sid/amd64/libspeechd-dev/filelist

Reply:
I feel like navit should be able to handle upstream behaviour. Or maybe there is some configure option for libspeechd that tells it to make those symlinks.

However if making those links is a de-facto standard among distributions, we should do it too. And we should notify upstream of that standard so that they can include the links in future releases.

Reply:
I still think this should be investigated, even though Mic92 merged already.